icons/myanimelist.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>MyAnimeList icon</title><path d="M9.675 11.217c-.04 2.392.62 4.732 1.382 6.783.577-.466 1.092-.89 1.614-1.329-.583-1.205-.81-2.619-.895-3.753l2.89.027v3.591h2.217V8.667h-2.217v2.306h-2.738c.042-3.062 3.715-2.924 5.308-2.89l-.5-1.984c-3.758-.393-7.01 1.24-7.06 5.118zm9.18 5.335h4.74L24 14.418h-3.11V6.065L18.855 6zM0 6.03v10.495h2.128V9.926l1.982 3.188 1.983-3.188v6.6h2.104V6.028H6.093L4.11 9.217 2.128 6.03z"/></svg>
```

Comment:
It looks like your icon (black) is a lot thinner than the actual logo underneath. Any chance you could take another look at this?

![image](https://user-images.githubusercontent.com/1830867/104452674-3b699880-559b-11eb-9543-248d33a8963c.png)

Reply:
Can you check this one?
I'm new to Inkscape and didn't use any other tools at all.